packages/webchat/src/components/OverridableComponent.tsx

```diff
@@ -32,7 +32,7 @@ class OverridableComponent extends React.Component<Props, State> {
     return this.props.overrides?.[this.props.name]
       ?.map(({ module, component }) => ({
         key: `${module}:${component}`,
-        element: window.botpress[module]?.[component]
+        element: (window.botpress as any)[module]?.[component]
```

Comment:
Why do you need to cast this as any?

Reply:
Otherwise it creates an implicit any. So we explicitly cast to any instead.